Remarkable-Donkeys/bangazon
# Employee and Customer Documentation

# Steps To Test #
1. git fetch --all
2. git checkout -b Tyler-General
3. Added Documentation for the following files: Employee.cs, Customer.cs, EmployeeComputer.cs, EmployeeController.cs, CustomerController.cs
4. Verify that all of the above files are commented with author, purpose, and method details.

### Description of what should happen when steps are followed ###
Verify that all of the above files are commented with author, purpose, and method details.

**At least one person needs to approve pull request before merged to master**
